app/src/ui/dialog/dialog-options.html

```diff
@@ -28,7 +28,8 @@
     static get properties() { return {
         message: { type: String, value: "" },
         open: { type: Boolean, value: false },
-        options: { type: Array, value: [$l("Dismiss")] }
+        options: { type: Array, value: [$l("Dismiss")] },
+        preventDismiss: { type: Boolean, value: true, reflectToAttribute: true }
```

Comment:
I understand that this was probably motivated by my comment about `preventDismiss = true ` being the 'default' in case of `pl-dialog-options` but still,  boolean attributes should always be `false` by default. I'd rather have it be set explicitly in `padlock.DialogMixin`.

`reflectToAttribute` is not really needed here, either.

Reply:
Ok, I see. But if I set `preventDismiss = true` in DialogMixin, will lack of attribute in `pl-dialog-options` override the explicit set to true? if yes, there is no point to set `preventDismiss = true` because every instance of `pl-dialog-options` is missing `prevent-dismiss` attribute and override this setting to false.

Reply:
Uhm, no. First of all, `DialogMixin` creates a `pl-dialog-options` element through `document.createElement`, not by parsing HTML. Second, of course setting the property explicitly will overwrite the initial default value, not the other way around.

Reply:
Ok, see. I'll try to fix it today. I was busy last week and apart from PC access.